sonar-java-plugin/src/main/java/org/sonar/plugins/java/bridges/DesignBridge.java

```diff
@@ -50,7 +50,7 @@
 
 public class DesignBridge {
 
-  private static final Logger LOG = LoggerFactory.getLogger(DesignBridge.class);
+  private static final Logger LOG = Loggers.get(DesignBridge.class);
 
   private final SensorContext context;
   private final DirectedGraph<Resource, Dependency> graph;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Resource"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "Dependency"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)